mico-core/src/main/java/io/github/ust/mico/core/dto/MicoServiceStatusDTO.java

```diff
@@ -69,9 +68,24 @@
      */
     private List<MicoServiceInterfaceDTO> interfacesInformation = new ArrayList<>();
 
+    /**
+     * List of {@link MicoUsingApplicationDTO}, representing all applications that are using one service together.
+     */
+    private List<MicoUsingApplicationDTO> usingApplications = new ArrayList<>();
+
     /**
      * List of {@link io.fabric8.kubernetes.api.model.Pod}s of a {@link io.github.ust.mico.core.model.MicoService}.
      */
     private List<KubernetesPodInfoDTO> podInfo = new ArrayList<>();
```

Comment:
Two remarks:
* You are using different naming conventions for the lists, rename them so they are more consistent:
   * `interfacesInformation`
   * `usingApplications`
   * `podInfo` → e.g. `podsInformation` / `podsInfo`
* It would be useful to make clear in the comment of `podInfo` that it includes all pods of all replicas of a deployment of a `MicoService`.

Reply:
podInfo is renamed to podsInformation, the comment is adapted